dist/common/scripts/scylla_coredump_setup

```diff
@@ -99,6 +99,7 @@ WantedBy=multi-user.target
         try:
             run('coredumpctl --no-pager --no-legend info {}'.format(pid))
             print('\nsystemd-coredump is working finely.')
+            run('rm /var/lib/systemd/coredump/core.bash.*')
```

Comment:
I think it is better to locate the file to delete the way cluster.py at sdcm does it.
see _notify_backtrace function.